### DISCUSSION TITLE
Admin user access

### DISCUSSION THREAD

**Discussion by diegoguerrad** [semaphoreci/semaphore#239] [Documentation improvements]
https://docs.semaphoreci.com/using-semaphore/rbac#org-admin

- Documentation states: They [admins] do not have access to billing information
- Documentation also states: View and manage billing plans

Currently Admins can see the Plans & Biling page, and from there should also have access to FS subscriptions. Opening this discussion to check if the first point "do not have access to billing information" is related to something different so we can clarify it or if we just need to update the docs.

**Comment by TomFern** [comment]
You're right, the text is contradicting, first it says that admins cannot access billing information, but later it says it does (in the bullet points). Can anyone confirm that admins can indeed view billing information and manage billing plans?
